Bot.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import telebot
+import ORMForBot as ORM
+from telebot import types
+import TicTacToeForBot as TicTacToe
+bot = telebot.TeleBot('5243687518:AAEYQJGdmCx86OvyIRTzAjDXwN1lM0dgFXU');
+nick = ''
+winner = 2
+theBoard = [' ']*10
+playerLetter, computerLetter = ['X', 'O']
+@bot.message_handler(content_types=['text'])
+
+def start_bot(message):
+    if message.text == "/start":
+        bot.send_message(message.from_user.id, "Hello, welcome to Tic-Tac-Toe! Please, enter your nickname.")
+        bot.register_next_step_handler(message, nickname)
+        
+def nickname(message):
+    global nick
+    nick = message.text
+    if ORM.check_name(nick):
+        ORM.write_player(nick)
+        bot.send_message(message.from_user.id, "Nice to meet you, "+ nick +"!\nTo play, type /play\nTo review your stats, type /stats\nIf you need help, type /help")
```

Comment:
It's better to make a shorter line, maybe to use multiline string:
```
"""
text
text
"""
```
and to use f-string to add a nick to it

Same comments for strings below

Reply:
Done